pkg/config/config_template.yaml

```diff
@@ -3983,3 +3982,72 @@ api_key:
   #
   # log_file: <TRAY_LOG_FILE_PATH>
 {{end -}}
+{{end -}}
+
+
+{{- if .APMInjection -}}
```

Comment:
should we bother checking if the OS is windows? or just assume that APM injection will only be enabled for windows?